packages/dev/core/src/Materials/Node/Blocks/Input/inputBlock.ts

```diff
@@ -395,6 +395,12 @@ export class InputBlock extends NodeMaterialBlock {
                 }
                 break;
             }
+            case AnimatedInputBlockTypes.MouseInfo: {
+                if (this.type === NodeMaterialBlockConnectionPointTypes.Vector4) {
+                    this.value = (window as any).mouse_data || new Vector4(0, 0, 0, 0);

```

Comment:
This is not ok as this will only work in the NME preview window. We need to get the data from the scene.inputmanager